asv/machine.py

```diff
@@ -53,8 +53,7 @@ def load(cls, machine_name, _path=None):
                 return d[list(d.keys())[0]]
 
         raise util.UserError(
-            "No information stored about machine '{0}'. I know about {1}.".format(
-                machine_name, util.human_list(d.keys())))
+            f"No information stored about machine '{machine_name}'. I know about {util.human_list(d.keys()))}."
```

Comment:
```suggestion
            f"No information stored about machine '{machine_name}'. I know about {util.human_list(d.keys()))}.")
```

You miss an `)` to close `util.UserError( )`.